### PR TITLE
creating initial structure of contrast adr

### PR DESCRIPTION
**Pull request by Gregory-Pereira**
Action item from DevSecOPs WG meeting Feb 9th, 2022. Initial pull request should only contain structure and general doc information, content to be added in following PRs.

### CODE REVIEW

File: .prow.yaml [hunk 1]
```diff
@@ -1,20 +1,13 @@
...
-              requests:
-                memory: "500Mi"
-                cpu: "300m"
-              limits:
-                memory: "1Gi"
-                cpu: "300m"
+    decorate: true
+    skip_report: false
+    always_run: true
+    context: aicoe-ci/prow/pre-commit
```
Comment (harshad16) [inline .prow.yaml]:
```suggestion
    context: pre-commit
```

In open-service-group its pre-commit :sweat_smile:

**Comment by Gregory-Pereira** [comment]
`pre-commit` will not run because the `.pre-commit-config.yaml` file merged in #6 is not updated. I have updated it in this PR even though it might be just barely out of scope. Pre-commit runs fine locally with the configured `.pre-commit-config.yaml` file, so dont let that hold the pr:
``` 
$ pre-commit run -a
Trim Trailing Whitespace..................................................Passed
Check for merge conflicts................................................Passed
Fix End of Files................................................................Passed
Check for added large files..............................................Passed
check BOM - deprecated: use fix-byte-order-marker........Passed
Check for case conflicts...................................................Passed
Check Yaml......................................................................Passed
Check for broken symlinks...............................................(no files to check)Skipped
Detect Private Key............................................................Passed
```
bump @HumairAK, @4n4nd, @harshad16, @Gkrumbach07

**Comment by Gregory-Pereira** [comment]
CI changes no longer live in this pr. Once #13 gets merged, builds should succeed.

**Comment by Gregory-Pereira** [comment]
/retest
also @HumairAK and @harshad16 do we still even need this ADR?

**Review by harshad16 — APPROVED** [review]
/ok-to-test
/lgtm
/approve

thanks :100:

**Pull request merged**